src/audio/drc/tune/sof_drc_paths.m

```diff
@@ -0,0 +1,24 @@
+function sof_drc_paths(enable)
+
+% sof_drc_paths(enable)
+% enable - set to 1 to enable needed search path
+%          set to 0 to disable the search paths
+%
+
+% SPDX-License-Identifier: BSD-3-Clause
+%
+% Copyright (c) 2024, Intel Corporation. All rights reserved.
+
+	common = '../../../../tools/tune/common';
+	% After #9187 use this
+	%eq = '../../eq_iir/tune';
+	% and remove this
+	eq = '../../../../tools/tune/eq';
+	if enable
+		addpath(common);
+		addpath(eq);
+	else
+		rmpath(common);
+		rmpath(eq);
+	end
+end
```

Comment:
This is fine for the moment (short term), but in the long term we need a script that sets the Matlab/Octave environment and paths up (we have a source script for Zephyr today- so something similar). See https://docs.octave.org/v4.2.1/Startup-Files.html as these mechanism can be used